src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/UniqueAltReadCount.java

```diff
@@ -11,31 +11,34 @@
 import org.broadinstitute.hellbender.engine.ReferenceContext;
 import org.broadinstitute.hellbender.utils.genotyper.ReadLikelihoods;
 import org.broadinstitute.hellbender.utils.help.HelpConstants;
+import org.broadinstitute.hellbender.tools.walkers.markduplicates.MarkDuplicatesGATK;
 
 import java.util.*;
 import java.util.stream.Collectors;
 
 /**
- * This annotation counts, at each variant locus, the number of ALT reads with unique start positions and fragment lengths.
- * i.e. multiple reads with the same start position and fragment length are grouped and counted only once as they are
- * likely duplicates. We have seen that the majority of false positives in low allele fraction, cell-free DNA samples
- * have the profile where the evidence for alternate allele comes solely from one, two, or three sets of apparent PCR-duplicates.
- * Reads in such a set have the same read-start and mate-end position i.e. they come from the same original insert.
- * Normally, they are marked as duplicates by MarkDuplicatesGATK. But when we use unique molecular identifiers (UMIs),
- * these apparent PCR-duplicate reads may get different UMIs, and hence to Mutect these reads appear as
- * independent evidence for ALT allele.
+ * Counts the number of unique pairs of (start positions, fragment length) among all alt reads.
```

Comment:
This is tricky because this is more stringent than uniqueness.  It groups together reads with different UMIs, which ought to mean that they are unique, having arisen from different molecules.  It does so out of caution that UMIs could have switched and so they are not really unique.

Perhaps I could say that it finds a lower bound on the number of unique reads?